src/node.h

```diff
@@ -23,31 +23,31 @@
 #define SRC_NODE_H_
 
 #ifdef _WIN32
-# ifndef BUILDING_NODE_EXTENSION
-#  define NODE_EXTERN __declspec(dllexport)
-# else
-#  define NODE_EXTERN __declspec(dllimport)
-# endif
+#ifndef BUILDING_NODE_EXTENSION
+#define NODE_EXTERN __declspec(dllexport)
 #else
-# define NODE_EXTERN __attribute__((visibility("default")))
+#define NODE_EXTERN __declspec(dllimport)
+#endif
+#else
+#define NODE_EXTERN __attribute__((visibility("default")))
```

Comment:
I'm not really a fan of losing the nested spaces here as they make the code far more readable in my opinion. Can't remember what our style guidelines on it say, however.

Reply:
I agree, we should keep these indented one way or another.

Reply:
The style guide doesn't mention indentation, is there a generally accepted unwritten rule for this that I can implement?